services/QuillLMS/app/services/clever_integration/classroom_students_importer.rb

```diff
@@ -0,0 +1,45 @@
+module CleverIntegration
+  class ClassroomStudentsImporter
+    attr_reader :classroom, :students_data
+
+    def initialize(classroom, students_data)
+      @classroom = classroom
+      @students_data = students_data
+    end
+
+    def run
+      return if students_data.nil? || students_data.empty?
```

Comment:
Rails has a shorter version of this (you can take or leave)
```suggestion
      return if students_data.blank?
```